digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -593,7 +614,7 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
     * @param roadAddressProject Updated road address project case class
     * @return Updated project reloaded from the database
     */
-  def saveProject(roadAddressProject: RoadAddressProject): RoadAddressProject = {
+  def saveProject(roadAddressProject: RoadAddressProject, resolution: Int = 8): RoadAddressProject = {
```

Comment:
Can the instances of resolution: Int = 8 replaced by a constant something like "resolution: Int = defaultZoomLevel"